src/Commands/RemoteCommand.php

```diff
@@ -64,11 +64,9 @@ protected function displayOutput($type, $line): void
 
             if ($type == Process::OUT) {
                 $this->output->write(trim($line) . PHP_EOL);
-
-                return;
+            } else {
```

Comment:
I prefer not to use `else` statements.

Would `continue` also work instead of this?

Reply:
You're right. I changed to use `continue`.